### ISSUE TITLE
Doctrine database configuration error

### ISSUE THREAD

**Issue by AscanderS** [symfony/symfony-docs#6691]
This is a discrepancy between the Symfony book and Symfony version 2.8 (at least) which is very disturbing for someone trying Symfony with PostgreSQL for the first time.

On page http://symfony.com/doc/2.8/book/doctrine.html It is explained that setting database_driver on app/config/parameters.yml allows to choose a driver.

```
# app/config/parameters.yml
parameters:
    database_driver:    pdo_mysql

```
However, parameter database_driver is not used (at least in version 2.8) since in app/config/config.yml you find that pdo_mysql is wired:

```
# app/config/config.yml
doctrine:
    dbal:
        driver:   pdo_mysql
```
There is no intention to add a database_driver parameter (see https://github.com/symfony/symfony/issues/19196), so please correct the corresponding page on the symfony book.

**Comment by iltar** [comment]
Fixed by #6689 (as mentioned in your previous issue)

**Comment by AscanderS** [comment]
Thanks. I hope I'll eventually appear fixed on the Symfony Book from version 2.8 (at least) on. 
Not the case for now.

**Comment by xabbuh** [comment]
The fix has been merged and will soon be visible on symfony.com.
